packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -208,13 +210,10 @@ class DDSTabsExtended extends MediaQueryMixin(
           `${ddsPrefix}-tab:nth-of-type(${_activeTabIndex + 1})`
         )?.shadowRoot?.querySelector(`.${prefix}--accordion__heading`);
 
-    if (activeItemControl instanceof HTMLElement) {
-      if (!isLargeOrGreater) {
-        // Unset focus so that when element is focused programmatically, the
-        // browser scrolls element into view.
-        activeItemControl.blur();
-      }
-      activeItemControl.focus();
```

Comment:
This change is what causes the regression Andy mentioned, @bruno-amorim . We need to programmatically set focus on the active item so the browser scrolls it into view.